src/cmd-buildextend-live

```diff
@@ -59,6 +59,12 @@ builddir = builds.get_build_dir(args.build)
 buildmeta_path = os.path.join(builddir, 'meta.json')
 buildmeta = GenericBuildMeta(workdir=workdir, build=args.build)
 
+# Build rhcos live iso with '--fast' will exit as not support lz4
+base_name = buildmeta['name']
+if base_name == "rhcos" and args.fast:
+    print("'--fast' is not supported for RHCOS, it is for FCOS")
```

Comment:
```suggestion
base_name = buildmeta['name']
if base_name == "rhcos" and args.fast:
    print("'--fast' requires LZ4 compressed squashfs support in the kernel. This is currently only available for FCOS")
```

Let's include the entire comment into the error message so that we keep all the information in one place?

Reply:
Good suggestion!